DESCRIPTION

```diff
@@ -41,7 +41,7 @@ Imports:
     magrittr (>= 1.5),
     tibble (>= 1.1),
     tidyr (>= 0.7.1),
-    readr (>= 1.0.0),
+    readr (>= 1.3.1),
```

Comment:
I was going going to comment about if bumping the required version was really necessary but after talking to you today I realize it is (to ensure consistency for OLD/NEW tests).  We should make sure we bump the version in `gcamdata.compdata` as well.

Reply:
I've done this.